app/containers/ErrorBoundary/ErrorBoundaryPage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+
+const ErrorBoundaryPage = props => (
```

Comment:
Let's reserve Page suffix for actual page containers. Pls rename, e.g. ErrorBoundryMessage